ps_core/ps_image_to_uvf.pro

```diff
@@ -57,20 +60,21 @@ pro ps_image_to_uvf, file_struct, n_vis_freq, kx_rad_vals, ky_rad_vals, $
       endif
       if n_elements(freq_flags) ne 0 then begin
         full_uvf_file = strjoin(strsplit(full_uvf_file, '_flag[a-z0-9]+', /regex, /extract))
+        full_uvf_file = strjoin(strsplit(full_uvf_file, '_flag+', /regex, /extract))
```

Comment:
I'm sure this works because you've tested it, but it confuses me. Can you add a comment explaining why there are two similar but different steps here?

Reply:
If you don't supply an optional `freq_flag_name` then the tag will be `_flag_` rather than `_flag<name>_`. 

We should just remove line 62 since line 63 makes it redundant.

Reply:
ok. I can take it out, can you rerun your tests when I do to make sure it doesn't muck anything up? I'm a little worried that the regex I wrote before was carefully tuned to handle something in particular but I don't recall what it was...